Directory.Build.props

```diff
@@ -55,6 +55,14 @@
     <Reference Include="Moq" />
   </ItemGroup>
 
+  <!-- Always copy content items into the build output directory. -->
```

Comment:
This doesn't happen "always". It only happens for test projects and even then the `<ItemDefinitionGroup>` only provides defaults.